README.md

```diff
@@ -1,28 +1,41 @@
-# HTML/CSS & JavaScript capstone project - Conference page
+# HTML/CSS & JavaScript capstone project 
 The aim of this project is putting the student in a real life situation where they are given a real-world-like project with specifications and a time duration to convert those specifications into a functional product.
 <br>
 My project was to replicate the Creative Commons 2015 Global Summit Conference site. 
 I was tasked to build the a minimum of 3 pages at two different screen sizes :
 - Mobile up to 768px
 - Desktop from 768px and bigger.
```

Comment:
As suggested by the first reviewer, kindly update your readme file using the [new template](https://github.com/microverseinc/readme-template) by Microverse. This is required in order to receive a review henceforth. Kindly check the notification channel on slack for more information about the template.